### PR TITLE
Add release notes for 2.10.8 and 2.11.1

### LINKED ISSUE #3711: Add changelogs for the 2.10.8 and 2.11.1 versions
**Product:** Tarantool
**Since:** 2.10.8/2.11.1
**Root document:**
- https://www.tarantool.io/en/doc/latest/release/2.11_series/
- https://www.tarantool.io/en/doc/latest/release/eol_versions/
- https://www.tarantool.io/en/doc/latest/release/calendar/#release-lifetime-table

# Details
- https://github.com/tarantool/tarantool/releases/tag/2.11.1
- https://github.com/tarantool/tarantool/releases/tag/2.10.8

# Definition of done

- [ ] Add a link to a GitHub release page for 2.11.1, rephrase the introduction sentence: https://www.tarantool.io/en/doc/latest/release/2.11_series/
- [ ] Create a new 2.10.8 page here: https://www.tarantool.io/en/doc/latest/release/eol_versions/
- [ ] Update the release calendar: https://www.tarantool.io/en/doc/latest/release/calendar/

### PR DESCRIPTION
**Pull request by andreyaksenov**
- https://docs.d.tarantool.io/en/doc/add-release-notes/release/2.11.0/ - moved to a top level, added links to GitHub releases
- https://docs.d.tarantool.io/en/doc/add-release-notes/release/2.10.8/
- https://docs.d.tarantool.io/en/doc/add-release-notes/release/calendar/

### CODE REVIEW

File: doc/release/2.10.8.rst [hunk 1]
```diff
@@ -0,0 +1,204 @@
...
+   ``box.ctl.promote()`` on a cluster with nodes configured with
+   ``election_mode = "candidate"`` (:tarantool-issue:`8497`).
+-  Fixed nodes configured with ``election_mode = 'candidate'``
+   spuriously detecting a split-vote when another candidate should win
+   with exactly a quorum of votes for it (:tarantool-issue:`8698`).
+
+LuaJIT
+~~~~~~
+
+Backported patches from the vanilla LuaJIT trunk (:tarantool-issue:`8516`).
```
Comment (p7nov) [inline doc/release/2.10.8.rst]: Two lists of LuaJIT backport can be merged

File: doc/release/calendar.rst [hunk 2]
```diff
@@ -34,49 +34,59 @@ after the specified date.
             -   End of life
             -   End of support
 
+        *   -   `2.11.1 LTS <https://github.com/tarantool/tarantool/releases/tag/2.11.1>`_
```
Comment (p7nov) [inline doc/release/calendar.rst:37]: Please add the new versions to the calendar diagram

File: doc/release/2.11_series.rst
Comment (p7nov) [inline doc/release/2.11_series.rst:1]: This page looks inconsistent and incomplete: we use the same link (`Tarantool 2.11.0/1`) text to links to different content and even different websites.
I suggest to improve it:
1. Rename the **Tarantool 2.11.0** page into **What's New in Tarantool 2.11** and change the link text accordingly.
2. Show the complete list of changelogs, including 2.11.0
3. Write explicitly that these are links to detailed changelogs, not just list of releases.
So the text above the list can be something like: `Below is a list of detailed changelogs for Tarantool 2.11 releases on GitHub`

**Review by p7nov — COMMENTED** [review]
Some ideas on improving the 2.11 release page.

File: doc/release/2.10.8.rst [hunk 3]
```diff
@@ -0,0 +1,204 @@
...
+
+-  Fixed console ignoring ``-i`` flag in case stdin is not a tty
+   (:tarantool-issue:`5064`).
+
+Datetime
+~~~~~~~~
+
+-  Fixed a bug raising a false positive error when creating new
+   intervals with range boundary values (:tarantool-issue:`8878`).
+-  Fixed a bug with buffer overflow in tnt_strptime (:tarantool-issue:`8502`).
```
Comment (xuniq) [inline doc/release/2.10.8.rst]:
```suggestion
-  Fixed a bug with buffer overflow in ``tnt_strptime`` (:tarantool-issue:`8502`).
```

File: doc/release/2.11_series.rst
Comment (andreyaksenov) [reply doc/release/2.11_series.rst:1]: Moved https://docs.d.tarantool.io/en/doc/add-release-notes/release/2.11.0/ to a top level, listed all 2.11 GitHub releases in one place.

**Review by p7nov — APPROVED** [review]

**Pull request merged**